### PR TITLE
monero-core_ru.ts

### PR DESCRIPTION
**Pull request by LvMsterfild**
russian language

### CODE REVIEW

**Comment by NanoAkron** [comment]
Can you merge #713 into this PR and squash please?

**Comment by LvMsterfild** [comment]
explain me how to do it...i am newbie...never worked with github

2017-05-05 1:52 GMT+03:00 Nano Akron <notifications@github.com>:

**Comment by NanoAkron** [comment]
@LvMsterfild Ok, best way is going to be:

`git checkout patch-1` <-- whatever you called this patch

Then make the change in this patch which you made in the other patch

Then `git add .` and `git commit` and just give it a simple commit message like `asdf`

Then `git rebase -i HEAD~2` which interactively rebases the last 2 commits (the original one, and the new one you've just named `asdf`)

Then next to the commit called `asdf` change `pick` to `squash` and save

Then it will ask you to change to commit message. Just delete the lines for the `asdf` commit and keep the original commit message for this patch (`russian language`) and save

Nearly there!

Now, `git push <name of your remote repo> --force` which forces the remote repo to accept the local change and overwrite it. This will automatically update this PR #712.

Then just delete PR #713 

I know it sounds like a lot, but it's quite fluid when you get into it. However, that's the limit of my git skills :)

**Comment by NanoAkron** [comment]
Hi @LvMsterfild unfortunately you've now mixed in more commits than before.

Best way to fix is:

git checkout master

git branch new

git checkout new

git cherry-pick XXX

Do that last one N times, with XXX as the hash for all the commits you need, in order

git rebase -i HEAD~N

replace the "pick" with "f", except the first one, which you replace by "r"

Edit commit message in the editor that comes up

Save, done.

File: translations/monero-core_ru.ts [hunk 1]
```diff
@@ -11,93 +11,93 @@
...
         <location filename="../pages/AddressBook.qml" line="61"/>
         <source>Address</source>
-        <translation type="unfinished"></translation>
+        <translation>Адрес</translation>
     </message>
     <message>
         <location filename="../pages/AddressBook.qml" line="63"/>
         <source>&lt;b&gt;Tip tekst test&lt;/b&gt;</source>
-        <translation type="unfinished"></translation>
+        <translation>&lt;b&gt;Текстовая подсказка...тест&lt;/b&gt;</translation>
```
Comment (jonathancross) [inline translations/monero-core_ru.ts]: This is nonsense placeholder text.  I'd suggest we **not** translate this so it is clear it must be updated eventually.
"Текстовая подсказка...тест" might be mistaken for something potentially useful in Russian.

Same for "Tip test test" and other placeholders below.

**Comment by LvMsterfild** [comment]
ok, I'll return it back...explain me (total newbie) please, where can I
find github console to correct all bad things that I've done....A man
before you give very good istructions, what to do...Maybe better somehow
delete all my commits and start it again, after editing ?

2017-05-10 19:06 GMT+03:00 Jonathan Cross <notifications@github.com>:

**Comment by jonathancross** [comment]
Hi @LvMsterfild , yes these things can be a bit complex  :-)

> where can I find github console...

There is no GitHub Console, @NanoAkron was suggesting that you run a series of `git` commands inside a command line terminal on your computer.  Seems you made these changes here on GitHub which explains why you don't know what he is talking about. Github is just a website that helps people share the work they do with git and do some simple edits.  The things he is suggesting cannot be done here on the website.

If you want to learn how to use git on the commandline, then great, but if not, then I can take what you did here and create a new Pull Request with it all squashed into a single commit, but then it will look like the changes are coming from me.

You can then review _my_ pull request for accuracy.

Sound good?

**Comment by LvMsterfild** [comment]
Of course. Do it. I'm just a noob :). Maybe some time later I'll improve my
skills with all that stuff. So "git" and "hub" are two different things). I
made many mistakes in translations. Later, I'll correct it and will do a
new commit.

2017-05-13 2:26 GMT+03:00 Jonathan Cross <notifications@github.com>:

**Pull request closed without merging**